lib/matplotlib/backends/backend_pgf.py

```diff
@@ -185,7 +205,7 @@ class LatexManager:
     @staticmethod
     def _build_latex_header():
         latex_header = [
-            r"\documentclass{article}",
```

Comment:
I think these could have been left as they were originally (but removing the 12pt below) no?  especially considering that we may want to make the documentclass configurable in the future.

Reply:
Not sure what you mean? These all have to be identical, so introducing a single name for them is just DRY. And in fact one of the first things one would do when making the documentclass configurable. I don’t see how it matters whether it’s a global constant or something else – that can always be changed with no effort. The point is to use a single variable for things that have to be the same.